editor/code_editor.cpp

```diff
@@ -765,6 +765,7 @@ void CodeTextEditor::update_editor_settings() {
 	text_editor->cursor_set_blink_enabled(EditorSettings::get_singleton()->get("text_editor/cursor/caret_blink"));
 	text_editor->cursor_set_blink_speed(EditorSettings::get_singleton()->get("text_editor/cursor/caret_blink_speed"));
 	text_editor->set_breakpoint_gutter_enabled(EditorSettings::get_singleton()->get("text_editor/line_numbers/show_breakpoint_gutter"));
+	text_editor->set_executing_line_gutter_enabled(EditorSettings::get_singleton()->get("text_editor/line_numbers/show_executing_line_gutter"));
```

Comment:
There shouldn't be a need for this setting - it can use the same one as breakpoints, as its shares the same gutter.

Reply:
Removed.